internal/validation/extension.go

```diff
@@ -238,3 +266,15 @@ func (s *Server) validate(r *http.Request, service *Service, operation *operatio
 		},
 	})
 }
+
+type ErrorBody struct {
+	Error string `json:"error,omitempty"`
+}
+
+func (e *ErrorBody) SetErrorBody(err error) {
+	errorMsg := ErrorBody{Error: err.Error()}
+	jsn, _ := json.Marshal(errorMsg)
```

Comment:
Are you intentionally ignoring the `error` from here?

Reply:
yes, as  there is no point in doing it. Either we get the error message or we don't